docs/pages/responsive-accordion-tabs.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Responsive Accordion Tabs
+description: Use the Markup of a Accordion or Tabs and create a Responsive Accordion Tabs.
```

Comment:
"Use the markup of the Accordion or Tabs components to create Responsive Accordion Tabs."